azure-pipelines.sds.master-release.yml

```diff
@@ -10,7 +10,7 @@ resources:
     - repository: azTemplates
       type: github
       name: hmcts/azure-devops-templates
-      ref: master
+      ref: apply-git-version-before-docker-build
```

Comment:
This needs to be master, that branch no longer exists. You only need to include fetchTags and the same applies to the pr release pipeline and the release pipeline

Reply:
thank you. Done